pinot-spi/src/main/java/org/apache/pinot/spi/utils/builder/ControllerRequestURLBuilder.java

```diff
@@ -239,6 +239,11 @@ public String forTableReload(String tableName, TableType tableType, boolean forc
     return StringUtil.join("/", _baseUrl, "segments", tableName, query);
   }
 
+  public String forTableNeedReload(String tableNameWithType) {
+    String query = String.format("needReload?verbose=%s", "false");
```

Comment:
(minor) No need to format

Reply:
Tested for false and true verbose flags. Changed this method